content/pt/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Controladores
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Em robótica e automação um _control loop_, ou em português _ciclo de controle_, é
+um ciclo não terminado que regula o estado de um sistema.
+
+Um exemplo de ciclo de controlo é um termostato de uma sala.
+
+Quando você define a temperatura, isso indica ao termostato
+sobre o seu *estado desejado*. A temperatura ambiente real é o
+*estado atual*. O termostato atua de forma a trazer o estado atual
+mais perto do estado desejado, ligando ou desligando o equipamento.
+
```

Comment:
Hey, looks like there is one missing phrase: "In Kubernetes, controllers are control loops that watch the state of your cluster, then make or request changes where needed. Each controller tries to move the current cluster state closer to the desired state." Can you check your translation againt the last revision of the docs?

Reply:
That's the glossary entry for the controller term. It is translated in the references.